packages/xod-client/src/editor/containers/Patch.jsx

```diff
@@ -95,21 +118,25 @@ class Patch extends React.Component {
   }
 
   onCommentResizeHandleMouseDown(event, commentId) {
+    if (isMiddleButtonPressed(event) || this.props.mode === EDITOR_MODE.PANNING) return;
```

Comment:
A lot of code duplication 😬 
But I haven't an idea how to fix it without dirty hacks, that could provide pain in the future (stop propagation at capturing phase for `onMouseDown`).